packages/block-library/src/table-of-contents/index.php

```diff
@@ -279,7 +279,7 @@ function ( $child_node ) use ( $entry_class, $page_url ) {
 				$child_node['children']
 					? block_core_table_of_contents_render_list(
 						$child_node['children'],
-						$page_url
+						esc_url( $page_url )
```

Comment:
@ZebulanStanphill Not sure if this is necessary but we're trying to follow practice of being zealous with escaping even if the variable is currently "known" and predictable. Let me know your thoughts. Much appreciated